sdk/eventgrid/azure-eventgrid/samples/sync_samples/sample_publish_events_to_a_topic_with_signature.py

```diff
@@ -4,13 +4,13 @@
 # license information.
 # --------------------------------------------------------------------------
 """
-FILE: cs1b_publish_custom_events_to_a_topic_with_signature.py
+FILE: sample_publish_custom_events_to_a_topic_with_signature.py
```

Comment:
rename file for clarity?
```suggestion
FILE: sample_publish_eg_events_to_a_topic_with_azure_sas_credential.py
```